scenarios/contact-menu.json

```diff
@@ -68,6 +68,50 @@
       "url": "{url}/civicrm/group?reset=1",
       "onReadyScript": "contact/manage-groups.js"
     },
+    {
+      "label": "Manage Groups - Edit Group settings modal",
+      "url": "{url}/civicrm/group?reset=1",
+      "selectors": [".ui-dialog"],
+      "onReadyScript": "contact/manage-groups-settings-modal.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts screen - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts to the group form - search form - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts-to-group-search-form.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts to the group form - results form - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts-to-group-results-form.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts to the group form - finalizing contacts to add - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts-to-group-finalizing-contacts.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts to the group form - view selected contacts - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "selectors": [".ui-dialog"],
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts-to-group-view-selected-contacts.js"
+    },   
+    {
+      "label": "Manage Groups - Add Contacts to the group form - Add New group Modal",
```

Comment:
this label doesn't seem to make sense. Adding a new group has nothing to do with adding contacts to a group. it should be called "Manage Groups - Add new group modal"